client/src/app/tabs/EngineProfile.js

```diff
@@ -240,6 +249,21 @@ function PlatformHint(props) {
   );
 }
 
+function UnknownVerisonHint(props) {
+  const {
+    displayLabel,
+    className
+  } = props;
+
+  return (
+    <div className={ className }>
+      This { displayLabel } diagram uses an unsupported version. Some features might not work as expected. To use the latest features, please <a href={ DONWLOAD_PAGE }>
```

Comment:
What we _could_ do here instead of opening the downloads page is to trigger an update check; if something is available the user will end up on "what's new" + a call to update.